### PR TITLE
docs: changed "deadbeef" to "cafebabe"

### PR DESCRIPTION
**Pull request by naz**
### Description

Changed use of resource hash example from "deadbeef" to "cafebabe".

### Motivation

My imagination is a bit too graphic and can imagine there are more people like me out there who might get triggered. Changed the sample to something a bit more positive ^_^

### Additional details

Reference of other hexspeak phrases - https://en.wikipedia.org/wiki/Hexspeak

### CODE REVIEW

**Comment by LPardue** [comment]
If this is going to be changed, I think it would be better to use a more inclusive value than the one currently suggested.

**Comment by Josh-Cena** [comment]
I won't argue that "CAFEBABE" doesn't age well today, but "CAFEBABE" and "DEADBEEF" are the only well-known hexspeaks. Changing it to something else would make it not much more meaningful than "12B9B0A1".

**Comment by LPardue** [comment]
The argument to include hexspeak at all is weak. This is an article about HTTP caching and ETags can contain any value specified in https://www.rfc-editor.org/rfc/rfc9110#name-etag, it isn't constrained to hex. The fact the current string doesn't include a `0x` prefix means the page is just confusing and distracting.

[RFC 9110 includes examples](https://www.rfc-editor.org/rfc/rfc9110#section-8.8.3-5) of ETag. MDN already has [many examples of ETags](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/ETag), I suggest reusing one of those.

File: files/en-us/web/http/caching/index.md [hunk 1]
```diff
@@ -233,14 +233,14 @@ To solve such problems, the `ETag` response header was standardized as an altern
 
 The value of the `ETag` response header is an arbitrary value generated by the server. There are no restrictions on how the server must generate the value, so servers are free to set the value based on whatever means they choose — such as a hash of the body contents or a version number.
 
-As an example, if a hash value is used for the `ETag` header and the hash value of the `index.html` resource is `deadbeef`, the response will be as follows:
+As an example, if a hash value is used for the `ETag` header and the hash value of the `index.html` resource is `cafebabe`, the response will be as follows:
```
Comment (wbamberg) [inline files/en-us/web/http/caching/index.md:236]:
```suggestion
As an example, if a hash value is used for the `ETag` header and the hash value of the `index.html` resource is `33a64df5`, the response will be as follows:
```

File: files/en-us/web/http/caching/index.md [hunk 2]
```diff
@@ -253,7 +253,7 @@ If that response is stale, the client takes the value of the `ETag` response hea
 GET /index.html HTTP/1.1
 Host: example.com
 Accept: text/html
-If-None-Match: "deadbeef"
+If-None-Match: "cafebabe"
```
Comment (wbamberg) [inline files/en-us/web/http/caching/index.md:256]:
```suggestion
If-None-Match: "33a64df5"
```

**Review by wbamberg — CHANGES_REQUESTED** [review]
Thanks for your PR, @naz !

I agree, there's no reason we need to use hexspeak here.

**Comment by naz** [comment]
Sorry, didn't mean to star a holy war :sweat_smile: It's nice to use cool names whenever possible to make things memorable. In here I don't think having any of the hexspeak phrases would benefit to the technical value of the document. Will change the values to a random hex proposed by @wbamberg :+1:

**Comment by schalkneethling** [comment]
Thank you for your contribution, and welcome to the community @naz 🎉

**Comment by LPardue** [comment]
Thanks @naz

File: files/en-us/web/http/caching/index.md [hunk 3]
```diff
@@ -233,14 +233,14 @@ To solve such problems, the `ETag` response header was standardized as an altern
...
-As an example, if a hash value is used for the `ETag` header and the hash value of the `index.html` resource is `deadbeef`, the response will be as follows:
+As an example, if a hash value is used for the `ETag` header and the hash value of the `index.html` resource is `33a64df5`, the response will be as follows:
 
 ```http
 HTTP/1.1 200 OK
 Content-Type: text/html
 Content-Length: 1024
 Date: Tue, 22 Feb 2022 22:22:22 GMT
-ETag: "deadbeef"
+ETag: "cafebabe"
```
Comment (wbamberg) [inline files/en-us/web/http/caching/index.md:243]:
```suggestion
ETag: "33a64df5"
```

**Review by wbamberg — APPROVED** [review]
👍 no worries @naz , and thanks for your PR!

**Pull request merged**